src/App.php

```diff
@@ -143,6 +145,8 @@ public function processRequest(ServerRequestInterface $request, float $start_tim
 			],
 		]);
 
+		$this->registerEventDispatcher();
```

Comment:
This would also need to be called in the other `process*` method, I assume this brevity is intended for the sake of the Proof of Concept?

Reply:
Yes, this is correct.